src/TestProblem/Hydro/KelvinHelmholtzInstability/Init_TestProb_Hydro_KelvinHelmholtzInstability.cpp

```diff
@@ -20,7 +21,9 @@ static double KH_Rho2;              // mass density in the lower region
        int    KH_RefineShearMaxLv;  // refine the regions around the shear plane to this level
        int    KH_PeriodicZFactor;   // decompose the simulation domain further into N periodic subdomains along the z axis
 
-static struct drand48_data drand_buf;
+uniform_real_distribution<double> distribution(0.0,1.0);
+default_random_engine generator;
```

Comment:
How about using `mt19937` instead of `default_random_engine` since the latter is implementation-dependent?